CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessPane.java

```diff
@@ -1019,7 +1018,12 @@ private void storeParameterValues() {
                         final Object obj = param.getValue().getObjectValue();
                         if (obj != null && obj.toString() != null && !obj.toString().isEmpty()) {
                             final String value = param.getValue().getStringValue();
-                            RecentParameterValues.storeRecentValue(id, value);
+                                final String typeValue = param.getValue().getType().toString();
+                            if (!typeValue.contains("LocalDateParameterType")) {
+                                    RecentParameterValues.storeRecentValue(id, value);
+                                } else {
+                                    RecentParameterValues.storeRecentValue(id, obj.toString());
+                                }
```

Comment:
This if-else can be converted to a tertiary statement:
`RecentParameterValues.storeRecentValue(id, !typeValue.contains("LocalDateParameterType") ? value : obj.toString())`